router/router_test.go

```diff
@@ -903,9 +934,9 @@ var _ = Describe("router", func() {
 			router.transformer = mockTransformer
 
 			router.enableBatching = true
-			router.reloadableConfig.noOfJobsToBatchInAWorker = 3
+			router.reloadableConfig.noOfJobsToBatchInAWorker = config.GetReloadableIntVar(3, 1, rand.UniqueString(10))
 			router.noOfWorkers = 1
-			router.reloadableConfig.routerTimeout = time.Duration(math.MaxInt64)
+			router.reloadableConfig.routerTimeout = config.GetReloadableDurationVar(math.MaxInt64, time.Nanosecond, rand.UniqueString(10))
```

Comment:
use `config.Set(...)`?

Reply:
I tried to perform the change "blindly" as much as possible to make the task easier. I will try to further optimise on the next task, where I will be introducing a `misc.SingleValueLoader`